packages/core/test/lib/envelope.test.ts

```diff
@@ -31,14 +31,16 @@ describe('createEventEnvelope', () => {
 
     const testTable: Array<[string, Event, EventTraceContext]> = [
       [
-        'adds only baggage item',
+        'adds only one baggage item',
```

Comment:
```suggestion
        'adds one baggage item',
```

Saying "only" one makes it seem like we're in danger of adding more and what we're testing is that we can restrict ourselves to just one, but I don't think that's the case here.